core/common/io/src/test/java/org/eclipse/rdf4j/common/io/ByteArrayUtilTest.java

```diff
@@ -72,24 +67,22 @@ public void testFindArray() {
 		assertEquals(-1, ByteArrayUtil.find(new byte[] { 0, 1, 2, 3, 4, 5, 6 }, 0, 20, new byte[] { 7, 8 }));
 	}
 
-	@Test
+	@Test(expected = ArrayIndexOutOfBoundsException.class)
 	public void testGetInt() {
 		assertEquals(0, ByteArrayUtil.getInt(new byte[] { 0, 0, 0, 0, 0, 0 }, 0));
 		assertEquals((1 << 24) + (2 << 16) + (3 << 8) + 4, ByteArrayUtil.getInt(new byte[] { 1, 2, 3, 4 }, 0));
 		assertEquals((1 << 24) + (2 << 16) + (3 << 8) + 4, ByteArrayUtil.getInt(new byte[] { 10, 1, 2, 3, 4, 10 }, 1));
 
-		thrown.expect(ArrayIndexOutOfBoundsException.class);
 		ByteArrayUtil.getInt(new byte[] { 1 }, 0);
 	}
 
-	@Test
+	@Test(expected = ArrayIndexOutOfBoundsException.class)
```

Comment:
Fwiw if you're doing JUnit upgrades any way, could we upgrade straight to JUnit 5? That's actually the preferred version now for any new tests - it's just that we have a lot of tests that haven't been upgraded yet... And if you're feeling _really_ generous, also have a look at using AssertJ assertions (though that one is a "preferred optional", not a must :) ).

Reply:
Good suggestion. I'll take a look. But I'll probably prioritize fixing Jackson before JUnit.